csunplugged/static/scss/website.scss

```diff
@@ -863,3 +863,21 @@ button:hover .icon-search {
         opacity: 0.5;
     }
 }
+
+.pii-navigation {
```

Comment:
Could you change pii here and below to `plugging-it-in` for consistency with a similar one further up?

Also maybe subclass this as `.btn.plugging-it-in-navigation` (if that's correct syntax) to make it clear this is for buttons